.travis.yml

```diff
@@ -45,6 +45,7 @@ deploy:
   - provider: s3
     on:
       tags: true
+      condition: $BUILD_OPENAPI_DOC=true
```

Comment:
I don't get the interest of putting a condition which is always true (env BUILD_OPENAPI_DOC is set to true in the env block at the beginning of .travis.yml)

Reply:
yes, this is true. I wanted to close it some days ago but I forgot. Hopefully you did not spend too much time on this one? :)